src/app/modules/submission/submission.utils.ts

```diff
@@ -23,14 +23,10 @@ type ResponseModeFilterParam = {
 
 // Exported for testing.
 export const getResponseModeFilter = (
-  responseMode: FormResponseMode,
+  encrypted: boolean,
```

Comment:
if boolean, the `isEncrypted` is better. Maybe `isEncryptedMode`?

Reply:
Done in [4bf6286](https://github.com/opengovsg/FormSG/pull/6595/commits/4bf6286022cba46dd59163f21e396c98fb21fbf5)! Hope the new names are good?